sdks/python/apache_beam/runners/worker/sdk_worker_test.py

```diff
@@ -23,7 +23,6 @@
 import logging
 import unittest
 from collections import namedtuple
-from typing import Any
```

Comment:
this change (minor internal pylint fix) makes the affected action to trigger: see a run on fork repo: https://github.com/Abacn/beam/actions/runs/5469847290/jobs/9959265164